src/themes/default/modules/transition.overrides

```diff
@@ -30,7 +30,7 @@
 
 /* In */
 @keyframes browseIn {
-  0% {
+  0 {
```

Comment:
This does (and all following changes in keyframes) not work anymore. keyframes have to be defined in percent, otherwise the value is completely ignored

Reply:
@lubber-de are you sure I did some testing and it seems to work 🤔

Reply:
yes, me too :wink:  try this:
http://jsfiddle.net/fdv6ajqs/

Using `0` does not work, chaning it back to `0%` shows the color fade animation

Reply:
Seems your right 😛, I will patch these tonight.